### ISSUE TITLE
Zs_RF_Shared.avsi sneo_FFT3D never uses neo_fft3d

### ISSUE THREAD

**Issue by Selur** [realfinder/AVS-Stuff#117]
Problem is that neo_fft3d does not have an 'ncpu' argument.
-> Zs_RF_Shared.avsi should be adjusted accordingly

**Comment by realfinder** [comment]
well, MeteorRain said it's added https://forum.doom9.org/showthread.php?p=1917423#post1917423

did you tried those https://forum.doom9.org/showthread.php?p=1923086#post1923086 ? IIRC sneo_FFT3D was work with them (since I don't use neo_FFT3D after I find this bug https://github.com/HomeOfAviSynthPlusEvolution/neo_FFT3D/issues/4)

**Comment by Selur** [comment]
That version does work with ncpu, but I think the script should not use it until there's a git release supporting it, or at least a comment should be added that the version from the doom9 link to solidfiles is needed.

**Comment by kedaitinh12** [comment]
The ver have ncpu here (compiled by realfinder):
http://www.solidfiles.com/v/55WKMW38dpX42

**Comment by Selur** [comment]
yes, the point is that the script should note that it only works with that version, since it is not the version folks will usually stumble upon.

**Comment by realfinder** [comment]
well, since there are bugs in neo one already I think using old one is better, until it fixed by msg7086

**Comment by realfinder** [comment]
this still problem after https://github.com/HomeOfAviSynthPlusEvolution/neo_FFT3D/releases ?
